docs/contributing/add_new_resource_tgc_next.md

```diff
@@ -0,0 +1,330 @@
+# Adding support for converting a new resource
+
+terraform-google-conversion (TGC) consumes a Terraform plan and uses it to build Cloud Asset Inventory (CAI) Assets. These built Assets only exist locally, in memory.
+
+TGC supports only those GCP resources that are available in both the Terraform provider and Cloud Asset Inventory.
+
+### Getting a terraform resource name from a GCP resource name
+
+The first step in determining if a GCP resource is supported is to figure out the name of the corresponding Terraform resource. You can often do this by searching for the GCP resource name in the [Terraform google-beta provider documentation](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs).
+
+### Getting the CAI asset type of a GCP resource
+
+The second step in determining if a GCP resource is supported is to verify its inclusion in the Cloud Asset Inventory (CAI). You can complete this by searching for the resource's asset type within CAI's list of [supported asset types](https://cloud.google.com/asset-inventory/docs/asset-types).
+
+## How to add support for a new resource
+
+To fully support a new resource in the terraform-google-conversion tool, you must implement support for it in both tfplan2cai and cai2hcl.
```

Comment:
This section should probably come after the section about adding the flag to the resource yaml.